featurizers/fingerprints.py

```diff
@@ -20,216 +21,133 @@
 class MorganFingerprint(FingerprintTransformer):
     def __init__(
         self,
-        radius: int = 2,
-        n_bits: int = 2048,
-        use_chirality: bool = False,
-        use_bond_types: bool = True,
-        use_features: bool = False,
+        radius: int = 3,
+        includeChirality: bool = False,
+        useBondTypes: bool = True,
+        onlyNonzeroInvariants: bool = False,
+        includeRingMembership: bool = True,
+        countBounds: AtomPairsParameters = None,
+        fpSize: int = 2048,
+        fingerprint_type: str = "bit",
```

Comment:
I think we should use Pythonic argument names, not ugly RDKit C++ style ones. After all, this is also why this library exists. And this is relatively small work, isn't it?

Reply:
For now changed to Optinal[List]. It's hard to decide whether this is correct as I wasn't able to find many example uses. for the generator. The ones I found typed countBounds as object and also initialized it as none